gapic/src/main/com/google/gapid/perfetto/models/VulkanEventTrack.java

```diff
@@ -51,6 +51,14 @@
   private static final String SLICES_SQL =
       "select " + BASE_COLUMNS + " from %s " +
           "where ts >= %d - dur and ts <= %d order by ts";
+  private static final String QUEUE_GROUP_START_SQL =
+      "select submission_id, min(ts) start from gpu_track t left join gpu_slice s " +
+          "on (t.id = s.track_id) where t.name like \'queue%%\' group by submission_id";
```

Comment:
This should probably find the correct tracks by scope rather than by name?

Reply:
Yes, please never rely on the name of the track (or almost anything for that matter) as an identifier.

Reply:
Thanks for pointing this out!